dom/nodes/insertion-removing-steps/Node-appendChild-form-and-script-from-fragment.tentative.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
```

Comment:
Technically these aren't using `appendChild` anymore I guess heh. Idk if we want to update the name of the test files or not, up to you. Just wanted to point that out since it feels a little redundant and is technically not accurate.

Reply:
Fixed